monkey/monkey_island/cc/services/zero_trust/zero_trust_report/pillar_service.py

```diff
@@ -0,0 +1,69 @@
+import common.common_consts.zero_trust_consts as zero_trust_consts
+from monkey_island.cc.models.zero_trust.finding import Finding
+
+
+class PillarService:
```

Comment:
Only in zero trust report resource

Reply:
I've already written about it, that "resources" are monkey island endpoints. Since the "services" aren't part of the resources, but are external to them, ZT services used directly by resources should have a ZT/SS prefix.

With that said, it's a bit of a gray area, so you can make the final call

Reply:
When reading code you can arrive at this code only from `ZeroTrustReport` resource. When reading code in `ZeroTrustReport` I think you'll end up with the same pattern for each if statement:
```
if (something) {
     return ZTSomething.some_data()
}
```
ZT in this case is redundant information IMO, especially since you're already in ZeroTrust report context. When it comes to imports it's the same redundancy, you do `from something.zero_trust import ZeroTrustFinding`. I think this issue can be solved even better if we didn't use classes for services, in that case we could use `zero_trust.finding.do_somethinng()`.